components/global/Node.vue

```diff
@@ -1,21 +1,17 @@
 <template>
   <div class="node">
-    <h2>Title: {{ title }}</h2>
+    <h2>Node: {{ title }}</h2>
     <div v-if="image" v-html="image.content" />
     <div v-if="body" v-html="body" />
-    <div v-if="sections">
```

Comment:
it seems this is something we should add to the default playground? this renders layout-builder if it's there but would be a good default